docs/build/CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+### Changelog
+
+All notable changes to this project will be documented in this file. Dates are displayed in UTC.
+
+Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+
+#### [1.2.0](https://github.com/comcast-sky/rdk-components-hal-doxygen/compare/1.1.0...1.2.0)
+
```

Comment:
Is there a date associated with this version, as included with previous versions?

Reply:
This isn't our file. This comes from the documentation template code.

Reply:
Got it!